deploy/operator/helm/templates/ingress.yaml

```diff
@@ -1,17 +1,32 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "spinnaker-operator.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- if and .Values.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
```

Comment:
ONE thing I've not tested - i thought capabilities was a hook and... if you're using spinnaker to try to do this it MAY not work...

Reply:
Hmm, good point. I think we've seen this before as well.

How would you feel about me just solidly switching over to the GA Ingress version, and then updating the compatibility matrix to say that whatever version this ends up in does not support kubernetes <`1.19`?

Reply:
Note, I'd just replace this entirely and remove the older non supported ingress versions ;) AND state that we only support k8s greater than 1.18.

Reply:
100% on "don't support k8s older than 1.19 ;) If that level of support is needed, the older releases work well.